docs/io/fits/usage/table.rst

```diff
@@ -130,6 +130,14 @@ the input table is generated::
     ...     hdu = fits.BinTableHDU(data=newdata)
     ...     hdu.writeto('newtable.fits')
 
+It is also possible to update in-place the data from the HDU object::
```

Comment:
That sounds odd. What is meant by in-place here? I mean it's actually written to a different file.

Reply:
It is related to the `hdu.data = hdu.data[mask]` line, which *updates in-place the data attribute*, which is different from creating a new HDU and was not supported in the past. Not sure what would be a better formulation ?